### PR TITLE
Add workflow config

### PR DESCRIPTION
**Pull request by wzxjohn**
Support change push site and detail config in workflow config.

#1

### CODE REVIEW

**Comment by pglombardo** [comment]
Nice PR - thanks @wzxjohn!  +1 on pointing to a privately hosted instance.

I'm without a laptop this week but I'll try to test this out as soon as I can, merge and republish.

File: info.plist [hunk 1]
```diff
@@ -105,11 +98,36 @@ require "net/http"
 require "uri"
 require "json"
 
-uri = URI.parse("https://pwpush.com/p.json")
+uri = URI.parse(ENV['site'] + "/p.json")
```
Comment (pglombardo) [inline info.plist]: It's a bit verbose, but let's use the [same environment variables](https://github.com/pglombardo/PasswordPusher/blob/5e87449439c9f282a45423a55c5b3f41bdd15f36/Configuration.md#proxies) as Password Pusher: `PWP__OVERRIDE_BASE_URL`

File: info.plist [hunk 2]
```diff
@@ -105,11 +98,36 @@ require "net/http"
...
+
+retrieval_step = "false"
+if ENV['retrieval_step'] == "1" then
```
Comment (pglombardo) [inline info.plist]: And let's do the same for the others.  Here are the environment variables:
https://github.com/pglombardo/PasswordPusher/blob/master/Configuration.md#password-push-expiration-settings

**Comment by wzxjohn** [comment]
All environment variables are changed. But Password Pusher does not have these two variable, so I just delete the `_DEFAULT` suffix from the most similar variable.
```
PWP__PW__EXPIRE_AFTER_DAYS
PWP__PW__EXPIRE_AFTER_VIEWS
```

**Comment by pglombardo** [comment]
Hi @wzxjohn - I haven't forgotten about this.  Just had a laptop die and got a replacement.  I'll revisit/merge this soon.  Thanks for the updates!

**Comment by pglombardo** [comment]
Thanks @wzxjohn!  I'll test this some more and try for a release soon.

**Pull request merged**